db/app.js

```diff
@@ -0,0 +1,19 @@
+
+const express = require('express')
+const { getTopicsController } = require('./controller/controller.js')
+const { handle404 } = require('./errorHandlers/errorHandlers.js')
```

Comment:
This handle 404 hasn't been used in this file yet
If it's handling a 404 endpoint not found, would be good to catch this using an `app.all`